src/st/np3/4B018.c

```diff
@@ -50,23 +62,126 @@ void func_801CD734() {
         func_801CD658();
 }
 
-INCLUDE_ASM("asm/us/st/np3/nonmatchings/4B018", func_801CD78C);
+void func_801CD78C(Entity* src, s32 speed, s16 angle, Entity* dst) {
+    if (g_CurrentEntity->facing != 0) {
+        angle = -angle;
+    }
+
+    //! FAKE:
+    (*(Point32*)dst) = (*(Point32*)src);
```

Comment:
I think it's more likely that x and y are the same struct like in this scratch https://decomp.me/scratch/9FZ5z

Reply:
Any way of doing it without modifying the Entity struct?

Reply:
@Xeeynamo Do you agree that this function suggests that the x and y pos were part of a struct inside Entity?

Reply:
That's pretty sure it is:
https://decomp.me/scratch/W0gtZ
https://decomp.me/scratch/yYULv

Reply:
We can, but, is it necessary? i mean it's already matching, if we're going to rebase the Entity struct we can change it then, otherwise it might get lost in the abbys, there's absolutely no problem for it being there don't you think?. Also i think these functions are unused, i couldn't find any call reference anywhere.